lib/karto_sdk/include/karto_sdk/Karto.h

```diff
@@ -4966,7 +4966,14 @@ namespace karto
         {
           kt_int32s index = GridIndex(gridIndex, false);
           T* pGridPointer = GetDataPointer();
-          pGridPointer[index]++;
+          //TODO:expose this via a .yaml
+          if( hitCntRemove ){
```

Comment:
I'm not sure why you're messing with the raycasting algorithm, can you explain this?

Reply:
Messing with the ray casting was necessary to interact with the cells which got passed (addint to the pass cnt) by the ray but not returned a hit. As that is the only way i saw to remove hits in a cell (removing a hitcnt).